runtime/Stdlib.res

```diff
@@ -53,9 +53,9 @@ type null<+'a> = Primitive_js_extern.null<'a>
 type undefined<+'a> = Primitive_js_extern.undefined<'a>
 type nullable<+'a> = Primitive_js_extern.nullable<'a>
 
-@val external window: Dom.window = "window"
-@val external document: Dom.document = "document"
-@val external globalThis: {..} = "globalThis"
+@deprecated("Use rescript-webapi instead") @val external window: Dom.window = "window"
+@deprecated("Use rescript-webapi instead") @val external document: Dom.document = "document"
+@deprecated("Use rescript-webapi instead") @val external globalThis: {..} = "globalThis"
```

Comment:
`globalThis` is not host-specific but is actually part of the ECMAScript spec: https://tc39.es/ecma262/multipage/global-object.html#sec-globalthis

So maybe the deprecation warning should say `Use Js.globalThis` instead of recommending `rescript-webapi`? (a [Js.globalThis](https://github.com/rescript-lang/rescript/blob/1d8560fcc75117024334f50a72f6bc0d3b507546/runtime/Js.res#L178-L180) binding already exists)

Reply:
Yes, actually `globalThis` was added by me :sweat_smile: 

It is intentional that it points to the `{..}` type. Depending on the environment, it can be a Node.js `global`, `DedicatedWorkerGlobalScope`, etc.

Others are pretty much host-specific

Reply:
So I should keep globalThis?

Reply:
I think so. It's part of the JS standard and useful when dealing with global variables or classes.

Here's a case: https://forum.rescript-lang.org/t/is-it-good-idea-having-binding-to-js-this/2922

But it should not be used too often.

Reply:
done